config.json

```diff
@@ -979,6 +979,17 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "proverb",
+      "uuid": "17ba80b8-1ca6-426b-880a-15de1d413032",
+      "core": false,
+      "unlocked_by": "pangram",
+      "difficulty": 4,
```

Comment:
regarding the order of placement in config.json - But I'd like to see this placed adjacent to the other difficulty 4 exercises. I guess these days with the tree-based structure, exactly where in the difficulty 4 exercises isn't important. But let's try keeping difficulties together unless we see another more valuable ordering (or it's clear that the difficulty-based ordering is detrimental)